src/go/k8s/apis/cluster.redpanda.com/v1alpha1/topic_types.go

```diff
@@ -291,7 +291,7 @@ func init() {
 	SchemeBuilder.Register(&Topic{}, &TopicList{})
 }
 
-func (t *Topic) GetName() string {
+func (t *Topic) GetTopicName() string {
```

Comment:
A bit redundant, is this to avoid collissions with something?

Reply:
Yes, collision with https://github.com/kubernetes/apimachinery/blob/839ad9438684f7ff469b04322abb28989149ad7f/pkg/apis/meta/v1/meta.go#L36